src/themes/light.scss

```diff
@@ -1,3 +1,10 @@
-.wxc-theme-light {
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-light {
   --wxc-meeting-background: linear-gradient(180deg, #F0F0F0 0%, #FFFFFF 100%);
+  --wxc-roster-participants-counter-color: $black-alpha-95;
```

Comment:
can you check if this is a standard color for titles? in which case the variable name should be less specific, eg: wxc-title-color

Also, I guess interpolation is needed here, too